dozer-ingestion/src/connectors/snowflake/connector/snowflake.rs

```diff
@@ -162,13 +173,28 @@ async fn run(
 
             info!("[{}][{}] Reading from changes stream", name, table.name);
 
-            consumer
-                .consume_stream(&stream_client, &table.name, ingestor, idx, iteration)
-                .await?;
+            consumer.consume_stream(&stream_client, &table.name, &ingestor, idx, iteration)?;
 
-            interval.tick().await;
+            std::thread::sleep(interval);
```

Comment:
This isn't an exact replacement. I need to find a blocking `interval.tick()`.

Reply:
Do we still need to sleep? I guess this `tick` was added to avoid blocking. @karolisg ?

Reply:
I think we need to sleep to avoid too frequent requests

Reply:
Got it